src/server/services/afis/afis-facturen.ts

```diff
@@ -260,15 +260,43 @@ function transformFacturen(
 ): AfisFacturenResponse {
   const feedProperties = getFeedEntryProperties(responseData);
   const count = responseData?.feed?.count ?? feedProperties.length;
-  const facturenTransformed = feedProperties.map((invoiceProperties) => {
-    return transformFactuur(invoiceProperties, sessionID, deelbetalingen);
-  });
+  const facturenTransformed = feedProperties
+    .filter((invoiceProperties) => {
+      return shouldFilterOutFactuur(invoiceProperties.PostingDate);
+    })
+    .map((invoiceProperties) => {
+      return transformFactuur(invoiceProperties, sessionID, deelbetalingen);
+    });
   return {
     count,
     facturen: facturenTransformed,
   };
 }
 
+function shouldFilterOutFactuur(postingDate: string): boolean {
```

Comment:
Ik denk dat je hier een specifieke naam moet kiezen. Bv `isDownloadAvailable()`

Reply:
Nu ben ik een beetje in de war, het ging om het helemaal wel of niet tonen van de factuur en niet om het downloaden toch? Even denken... Misschien iets als `displayFactuur()` ? of `isInDisplayTime()` ?

Of bedoel je te zeggen dat die niet getoont word omdat er geen download om die tijd beschikbaar is?

Reply:
Beschikbaar is:
- postDate vóór vandaag
- postDate vandaag en ná 19:00

De rest willen we dus niet.

Dus als je zeft: 

```
function isDownloadAvailable() {
  return postDate vóór vandaag OR postDate vandaag en ná 19:00
}
```

Reply:
Ah, yes gedaan